tools/filter_cells_and_genes/filter_cells_and_genes.xml

```diff
@@ -0,0 +1,439 @@
+<tool id="filter_cells_and_genes" name="Filter out cells and genes" version="0.1.0">
+  <description>from raw counts single cell RNAseq expression values"</description>
+  <requirements>
+      <requirement type="package" version="2.3.1=r3.4.1_0">r-seurat</requirement>
+      <requirement type="package" version="1.6.0=r3.4.1_0">bioconductor-scater</requirement>
+      <requirement type="package" version="1.0.0=r3.4.1_0">bioconductor-singlecellexperiment</requirement>
+   </requirements>
+   <stdio>
+      <exit_code range="1:" level="fatal" description="Tool exception" />
+   </stdio>
+   <command detect_errors="exit_code"><![CDATA[ Rscript $filtering_out ]]></command>
+   <configfiles>
+      <configfile name="filtering_out">
+<![CDATA[
+## Setup R error handling to go to stderr
+options( show.error.messages=F,  error = function () { cat( geterrmessage(), file=stderr() ); q( "no", 1, F ) } )
```

Comment:
@bellenger-l can you set a filter_cells_and_genes.R script call by the wrapper ? see https://github.com/ARTbio/tools-artbio/blob/master/tools/small_rna_maps/small_rna_maps.xml#L42 for an example

Reply:
Ok no problem !